packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -293,6 +295,37 @@ const Account = Backbone.Model.extend(
       );
     },
 
+    /**
+     * Given a user's `kB` and a pipeline public key chosen at random from the
+     * key bundle, derive the ecosystem_user_id and encrypt it with the
+     * pipeline key, yielding the ecosystem_anon_id. Return the
+     * ecosystem_anon_id.
+     *
+     * @param {String} kB - user's kB
+     * @param {String} pipelineKey - a randomly-chosen pipeline public key
+     * in JWK format
+     * @returns {Promise} A promise that will be fulfilled with the user's
+     * ecosystem_anon_id in JWK format
+     */
+    async generateEcosystemAnonID(kB, pipelineKey) {
+      const uid = this.get('uid');
+      const keyData = {
+        // This identifier comes from the 'Account Ecosysten Telemetry
+        // Identifiers' google doc.
+        identifier: 'https://identity.mozilla.com/ids/ecosystem_telemetry',
```

Comment:
BTW, I made this up more-or-less arbitrarily, bikeshedding most welcome on the identifier here; /cc @vladikoff

Reply:
Thoughts after staring at this for a bit:

I don't know if we'll put anything else under the `/ids/` namespace, but AET probably should stand alone in this sense.

I did notice that we have a dash-delimited URI in `https://identity.mozilla.com/cmd/open-uri`, so maybe for consistency, this scope should be `https://identity.mozilla.com/ids/ecosystem-telemetry`?